### PR TITLE
Create Os.py

### PR DESCRIPTION
**Pull request by ricky-aug**
Please do the review and review is assinged to 2 reviewers

### CODE REVIEW

File: Os.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#python defined packages
+import sys
+import argparse
```
Comment (ricky001) [inline Os.py:3]: Why argparse go with JSON it will be fast

File: Os.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+import os
+import json
+
+#external packages
```
Comment (ricky001) [inline Os.py:7]: Please make the document which shows how to install external packages

**Review by ricky001 — CHANGES_REQUESTED** [review]
Please address the review comments

**Review by tricharu — APPROVED** [review]
As such nothing found from my side

File: Os.py (hunk 1 above)
Comment (ricky-aug) [reply Os.py:3]: Changes Done

File: Os.py (hunk 2 above)
Comment (ricky-aug) [reply Os.py:7]: Sure will create the manual

**Review by ricky-aug — COMMENTED** [review]
Addressed the changes please approve

**Review by ricky001 — APPROVED** [review]
Looks Good will merge

**Comment by ricky-aug** [comment]
ok we will do later

**Pull request merged**